internals/core_models.py

```diff
@@ -94,6 +94,7 @@ class FeatureEntry(ndb.Model):  # Copy from Feature
   star_count = ndb.IntegerProperty(default=0)
   search_tags = ndb.StringProperty(repeated=True)
   feature_notes = ndb.TextProperty()  # copy from comments
+  web_feature = ndb.TextProperty()
```

Comment:
Why not StringProperty?  A TextProperty will not be index in NDB so we would not be able to query on its value.